File: 7 kyu/(7 kyu) Vowel count/(7 kyu) Vowel count.js
function getCount(str) {
  let vowelsCount = 0;

  for (char of str.split('')) {
    if (['a', 'e', 'i', 'o', 'u'].includes(char)) {
      vowelsCount++;
    }
  }

  return vowelsCount;
}

File: 7 kyu/(7 kyu) Vowel count/(7 kyu) Vowel count.dart
import "dart:core";

int getCount(String inputStr){
  //your code here
  var vowelsCount = 0;

  for (final char in inputStr.split('')) {
    if (['a', 'e', 'i', 'o', 'u'].contains(char)) {
      vowelsCount += 1;
    }
  }

  return vowelsCount;
}
